src/shared/services/map-location/index.js

```diff
@@ -1,28 +1,60 @@
-function mapLocation(loc) {
+export const location2feature = location => ({
+  type: 'Point',
+  coordinates: [location.lng, location.lat],
+});
+
+export const feature2location = feature => {
+  const { coordinates } = feature;
+  return {
+    lat: coordinates[1],
+    lng: coordinates[0],
+  };
+};
+
+export const address2pdok = address => {
+  const { openbare_ruimte, huisnummer, huisletter, huisnummer_toevoeging, postcode, woonplaats } = address;
+
+  return {
+    straatnaam: openbare_ruimte,
+    huisnummer: `${huisnummer}`,
+    huisletter: huisletter || '',
+    huisnummertoevoeging: huisnummer_toevoeging ? String(huisnummer_toevoeging) : '',
+    postcode,
+    woonplaatsnaam: woonplaats,
+  };
+};
+
+/**
+ * converts the location from `sia` location format to latlon format
+ */
+export function mapLocation(loc) {
```

Comment:
Please use `const`. Also, why is this function a named export as well as a default export? Please use either one of the two.

Reply:
Right! Changed to const.